src/asset-swapper/utils/market_operation_utils/constants.ts

```diff
@@ -2330,6 +2328,14 @@ export const BALANCER_V2_SUBGRAPH_URL_BY_CHAIN = valueByChainId(
     null,
 );
 
+export const BEETHOVEN_X_SUBGRAPH_URL_BY_CHAIN = valueByChainId(
+    {
+        [ChainId.Fantom]: 'https://api.thegraph.com/subgraphs/name/beethovenxfi/beethovenx',
+        [ChainId.Optimism]: 'https://api.thegraph.com/subgraphs/name/beethovenxfi/beethovenx-optimism',
```

Comment:
We need these endpoints for Balancer V2/BeethovenX because it allows us to query for Balancer pool metadata. We will then use this Balancer pool metadata on Balancer Labs' SOR to figure out multihop swap steps to sample for.